pkg/util/testing/runtime.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2024 The Kubeflow Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func ToObject(s *runtime.Scheme, objects ...any) ([]runtime.Object, error) {
+	var objs []runtime.Object
+	for _, obj := range objects {
+		if o, err := toObject(s, obj); err != nil {
+			return nil, err
+		} else {
+			objs = append(objs, o)
+		}
+	}
+	return objs, nil
+}
+
+func toObject(s *runtime.Scheme, obj any) (runtime.Object, error) {
+	if o, ok := obj.(runtime.Object); ok {
+		return o, nil
+	}
+	var u *unstructured.Unstructured
+	if o, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj); err != nil {
+		return nil, err
+	} else {
+		u = &unstructured.Unstructured{Object: o}
+	}
```

Comment:
```suggestion
	}
	u = &unstructured.Unstructured{Object: o}
```

Reply:
That's needed as `o` is in the if / else block scope.

Reply:
This was resolved in another thread.